pkg/elf/elf_test.go

```diff
@@ -197,7 +198,9 @@ func BenchmarkWriteELF(b *testing.B) {
 	defer os.RemoveAll(tmpDir)
 
 	elf, err := Open(baseObjPath)
-	if err != nil {
+	if errors.Is(err, os.ErrNotExist) {
```

Comment:
The recommendation as you know is to use fs.ErrNotExist but I guess to support more versions of Go you are using os.ErrNotExist. I think if we are in this direction we might as well just do os.IsNotExist(err) to handle any syscall edges too?

Reply:
It would be nice to also add a comment referencing the issue for posterity:

// If the ELF file couldn't be found most likely it
// wasn't built. See issue \<link>

Reply:
@odeke-em thanks for the feedback. If initially tried `os.IsNotExist` but that didn't seem to work. Guess we're hitting https://golang.org/issues/38198. I don't see a recommendation anywhere to prefer fs.ErrNotExist over os.ErrNotExist in the godoc or the Go release notes. Where does it say that the former is preferred?

I'll add that comment referencing the issue, good point.

Reply:
It is recommendEd at https://pkg.go.dev/os#IsNotExist
![image](https://user-images.githubusercontent.com/4898263/136174262-ee67e93e-599f-45ab-bebd-f30557ad7f48.jpeg)
Plus I was part of the reviews for the fs package hence the preference for it but what you have works, thank you!